content/_changelogs/11.0.1.md

```diff
@@ -0,0 +1,19 @@
+## 11.0.1
+
+_Released 11/09/2022_
+
+**Bugfixes:**
+
+- Fixed an issue that caused enabling experimentalSessionAndOrigin to throw a
+  webpack error. Fixes
+  [#24611](https://github.com/cypress-io/cypress/issues/24611)
+- Fixed an issue where using the mochawesome reporter would cause Cypress to
+  throw a 'Cannot find module' error. Fixes
+  [#24607](https://github.com/cypress-io/cypress/issues/24607)
+- Fixed testIsolation configuration validation to allow configuration updates
+  without restarting Cypress. Fixes
+  [#24497](https://github.com/cypress-io/cypress/issues/24497).
+- Fixed an issue where cy.origin would not prevent users from setting up
+  same-superdomain-origin cy.origin blocks. In these cases cy.origin is not
```

Comment:
```suggestion
  `same-superdomain-origin` `cy.origin()` blocks. In these cases `cy.origin()` is not
```

is same-superdomain-origin a Cypress term? If so, can we expand on what thtis is?